spec/features/note_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'capybara/rspec'
+include RequestHelpers
+
+feature 'Team leader creates a new note' do
```

Comment:
There are 3 features with the same name. Are they all the same?

Reply:
Resolved!